app/services/target_service/creator.rb

```diff
@@ -8,7 +8,9 @@ def initialize(targets, target_params)
     end
 
     def call
-      @targets.create!(@target_params)
+      target = @targets.create!(@target_params)
```

Comment:
The creation of the target should also go inside of the TargetService.

Reply:
The creation of the target is inside the TargetService::Creator. Where are you suggesting to move it? @sebastiancaraballo